dev/Pager/PrototypePager/PrototypePager.Properties.cs

```diff
@@ -0,0 +1,180 @@
+﻿using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace MUXControlsTestApp
+{
+    public sealed partial class PrototypePager : Control
+    {
+        public PagerDisplayModes PagerDisplayMode
+        {
+            get { return (PagerDisplayModes)GetValue(PagerDisplayModeProperty); }
+            set { SetValue(PagerDisplayModeProperty, value); }
+        }
+        public static readonly DependencyProperty PagerDisplayModeProperty =
+            DependencyProperty.Register("PagerDisplayMode", typeof(PagerDisplayModes), typeof(PrototypePager), new PropertyMetadata(PagerDisplayModes.Auto));
+
+        /*  Universal Pager Properties
+         * 
+         */
+        public PagerTemplateSettings PagerTemplateSettings
+        {
+            get { return (PagerTemplateSettings)GetValue(PagerTemplateSettingsProperty); }
+            private set { SetValue(PagerTemplateSettingsProperty, value); }
+        }
+        public static readonly DependencyProperty PagerTemplateSettingsProperty =
```

Comment:
should not expose the TemplateSettings property to follow the same pattern as other controls. This should be private (or internal)

Reply:
Done

Reply:
I think the pattern that I have been seeing that other controls follow are that the TemplateSettings property is a public read-only variable. I refactored the code to follow this structure.

Reply:
Right, but making the PagerTemplateSettingsProperty public allows one to set the value though
pager.SetValue(PagerTemplateSettingsProperty, myValue) even though the setter for the property is read only.

Reply:
Ok, I think I understand you now. I've updated the property.